janus-idp/backstage-showcase
# Add Tekton plugin

## Description

- Adds the Tekton plugin to the Showcase App. The CI tab of the App will display the latest pipeline run visualization if the tekton CI annotation (`janus-idp.io/tekton-enabled: 'true'`) is available.
- Updated the topology package version

## Which issue(s) does this PR fix

- Fixes https://github.com/janus-idp/backstage-showcase/issues/172

## PR acceptance criteria

Please make sure that the following steps are complete:

- GitHub Actions are completed and successful
- Unit Tests are updated and passing
- E2E Tests are updated and passing
- Documentation is updated if necessary
- Add a screenshot if the change is UX/UI related

## How to test changes / Special notes to the reviewer

- Add ${K8S_CLUSTER_NAME} , ${K8S_CLUSTER_URL} ,  ${K8S_CLUSTER_TOKEN} info
- Create a pipeline run in the above cluster and ensure that the resource has the "backstage.io/kubernetes-id= test-backstage" label

## Screenshots
<img width="1601" alt="Screenshot 2023-05-03 at 3 40 56 PM" src="https://user-images.githubusercontent.com/22490998/235888964-12a38bb6-e356-4603-a32b-52c62581b9b8.png">

<img width="1260" alt="Screenshot 2023-05-04 at 10 14 08 PM" src="https://user-images.githubusercontent.com/22490998/236271244-1d7463ce-72e1-4e55-b91e-683d59620b30.png">
